### PR TITLE
SC-2536: Cocoapods support

### PR DESCRIPTION
**Pull request by ericn441**
#### Purpose
- [x] Feature
- [ ] Bug Fix
- [ ] Hotfix
- [ ] Other

#### Associated Tickets
- [Unleash Cocoapods](https://silvercar.atlassian.net/browse/SC-2536)

#### Details
Adds cocoapods support to enable unleash for dw-ios

#### Checklist
- [] Tests
- [] Correct base and merge branches

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
```
Comment (FernandoX7) [inline Podfile:2]: Should this specify the latest platform version instead?

File: Podfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+install! 'cocoapods', :disable_input_output_paths => true
+
+
+target 'Unleash' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for Unleash
+  pod "PromiseKit", "~> 6.8"
```
Comment (louisvdaily) [inline Podfile:11]: Do we need to have this Podfile? Can we tell the `podspec` that a dependency this project has is `PromiseKit`?

File: Podfile (hunk 1 above)
Comment (ericn441) [reply Podfile:2]: Its minimum supported platform. This library is also open sourced so we can support platforms below what we support internally.

File: Podfile (hunk 2 above)
Comment (ericn441) [reply Podfile:11]: Its for the sample app

Comment (ericn441) [reply Podfile:11]: But we can remove it

Comment (ericn441) [reply Podfile:11]: Actually I'll remove it & update the readme

**Comment by deandalm** [comment]
I agree with Louis, we shouldn't be checking in source code of dependencies. If that is a limitation of Cocapods then we need to move back to Carthage and use wrappers to integrate 3rd party applications like we did with Firebase.

**Comment by deandalm** [comment]
@ericn441 the recent commit you made cleaned up how many files you were checking in, did you run into a problem?

**Pull request closed without merging**